test/transmuxer.test.js

```diff
@@ -3324,6 +3324,130 @@ QUnit.test('can specify that we want to generate separate audio and video segmen
   QUnit.equal('mdat', boxes[1].type, 'generated a mdat box');
 });
 
+QUnit.test("doesn't adjust caption and ID3 times when configured to adjust timestamps", function() {
```

Comment:
We may want a comment explaining why we have this case, since it stands out as odd just by reading it.

Reply:
ah, I think I made a mistake with the name of this 🤦‍♀️